mypyc/irbuild/prebuildvisitor.py

```diff
@@ -82,11 +87,20 @@ def __init__(
 
         self.current_file: MypyFile = current_file
 
+        self.missing_types_visitor = MissingTypesVisitor(types)
+
     def visit(self, o: Node) -> bool:
         if not isinstance(o, Import):
             self._current_import_group = None
         return True
 
+    def visit_assignment_stmt(self, stmt: AssignmentStmt) -> None:
+        # These are cases where mypy may not have types for certain expressions,
+        # but mypyc needs some form type to exist.
+        if isinstance(stmt.rvalue, IndexExpr) and stmt.rvalue.analyzed:
```

Comment:
A better check would be `if stmt.is_alias_def:`, since this would trigger for all aliases, and if the `IndexExpr` is nested.

Reply:
Thanks, I missed the `is_alias_def`!